cypress/e2e/ui/categorization-uitests.cy.js

```diff
@@ -0,0 +1,239 @@
+describe("tests on the categorization page via ui interaction", () => {
```

Comment:
UI-based tests now follow the same organization used by page-based/programmatic-state-based testing.